.github/workflows/canary-deploy.yml

```diff
@@ -0,0 +1,61 @@
+name: Canary Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: Canary Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@master
+      with:
+        # Canary release script requires git history and tags.
+        fetch-depth: 0
+    - name: Set up Node (10)
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - name: Yarn install
+      run: yarn
+    - name: Deploy canary
+      run: yarn release --canary
```

Comment:
skipTest?

Reply:
It already skips tests, cli script branches to canary release flow before tests are run.